File: ADDREV_Adding Reversed Numbers/42_Adding Reversed Numbers.R
# Project name : SPOJ: Adding reversed numbers
# Author       : Wojciech Raszka
# Date created : 2019-02-06
# Description  :
# Status       : Accepted (???)

library('stringi')

f <- file('stdin', open='r')

no_of_cases <- as.integer(readLines(f, n=1))

for (i in 1:no_of_cases){
  n <- as.integer(stri_reverse(as.character(sum(as.integer(stri_reverse(unlist(strsplit(readLines(f, n=1), ' '))))))))
  write(n, stdout())
}

File: ADDREV_Adding Reversed Numbers/42_Adding Reversed Numbers.py
# Project name : SPOJ: ADDREV -  Adding Reversed Numbers
# Author       : Wojciech Raszka
# E-mail       : gitpistachio@gmail.com
# Date created : 2019-??-??
# Description  :
# Status       : Accepted ()
# Tags         : python
# Comment      :

no_of_cases = int(input())

for c in range(no_of_cases):
    print(int(str(sum(map(lambda s: int(s[::-1]), input().split(' '))))[::-1]))
